pptx_blueprint/__init__.py

```diff
@@ -42,14 +42,57 @@ def replace_picture(self, label: str, filename: _Pathlike) -> None:
         """
         pass
 
-    def replace_table(self, label: str, data) -> None:
+    def replace_table(self, label: str, data, header=False, rownames=False):
         """Replaces rectangle placeholders on one or many slides.
 
         Args:
             label (str): label of the placeholder (without curly braces)
             data (pandas.DataFrame): table to be inserted into the presentation
         """
-        pass
+        assert isinstance(data, pandas.dataframe)
+        
+        shapes_to_replace = self._find_shapes(label)
```

Comment:
In #7, the method signature of `_find_shapes` will be changed? Maybe change this to reflect that?